examples/snmp/pfsense.js

```diff
@@ -0,0 +1,889 @@
+		order: 0
+	},
+	{
+		label: 'Source tracking table limit',
+		unit: '',
+		oid: '1.3.6.1.4.1.12325.1.200.1.5.2.0',
+		order: 0
+	},
+	{
+		label: 'Source tracking table utilization',
+		unit: '%',
+		oid: '__calculated_tracking_table_usage',
+		postProcess: [function (last, next) {
+			var count = get_result('1.3.6.1.4.1.12325.1.200.1.4.1.0');
+			var limit = get_result('1.3.6.1.4.1.12325.1.200.1.5.2.0');
+			this.result = (count * 100) / limit;
+			next(this);
+		}],
+		snmpget: false
+	},
+	{
+		label: 'State of nginx process',
+		unit: '',
+		walk: {
+			ref: '1.3.6.1.2.1.25.4.2.1.2',
+			name: 'nginx',
+			oid: '1.3.6.1.2.1.25.4.2.1.7',
+			callback: update_oid
+		}
+	},
+	{
+		label: 'States table current',
+		unit: '',
+		oid: '1.3.6.1.4.1.12325.1.200.1.3.1.0',
+		order: 0
+	},
+	{
+		label: 'States table limit',
+		unit: '',
+		oid: '1.3.6.1.4.1.12325.1.200.1.5.1.0',
+		order: 0
+	},
+	{
+		label: 'States table utilization',
+		unit: '%',
+		oid: '__calculated_states_table_usage',
+		postProcess: [function (last, next) {
+			var count = get_result('1.3.6.1.4.1.12325.1.200.1.3.1.0');
+			var limit = get_result('1.3.6.1.4.1.12325.1.200.1.5.1.0');
+			this.result = (count * 100) / limit;
+			next(this);
+		}],
+		snmpget: false
+	},
+	{
+		label: 'System contact details',
+		unit: '',
+		oid: '1.3.6.1.2.1.1.4.0'
+	},
+	{
+		label: 'System description',
+		unit: '',
+		oid: '1.3.6.1.2.1.1.1.0'
+	},
+	{
+		label: 'System location',
+		unit: '',
+		oid: '1.3.6.1.2.1.1.6.0'
+	},
+	{
+		label: 'System name',
+		unit: '',
+		oid: '1.3.6.1.2.1.1.5.0'
+	},
+	{
+		label: 'System object ID',
+		unit: '',
+		oid: '1.3.6.1.2.1.1.2.0'
+	},
+	{
+		'label': 'Uptime',
+		'unit': 'uptime',
+		'oid': '1.3.6.1.2.1.25.1.1.0'
+	},
+];
+
+function execute_all(arrayFn, callback) {
+	if (arrayFn.length == 0) {
+		callback([]);
+	}
+	var length = arrayFn.length;
+	var results = new Array(length);
+	var finished = 0;
+	arrayFn.forEach(function (fn, index) {
+		fn(function (result) {
+			results[index] = result;
+			if (++finished == length) {
+				callback(results);
+			}
+		});
+	});
+}
+
+function execute_seq(functions, callback) {
+	var _this = this;
+	var callbackResult = null;
+	function executeNext(functionIndex) {
+		if (functionIndex == functions.length) return callback.apply(_this, [callbackResult]);
+		functions[functionIndex].apply(_this, [callbackResult, function (result) {
+			callbackResult = result;
+			executeNext.apply(_this, [++functionIndex]);
+		}]);
+	}
+	executeNext.apply(_this, [0]);
+}
+
+function get_result(oid) {
+	return snmp_parameters.filter(function (p) {
+		return p.oid == oid;
+	}).map(function (p) {
+		return p.result;
+	})[0];
+}
+
+function update_oid(param, res) {
+	for (var key in res) {
+		if (res[key] == param.walk.name) {
+			var oidArray = key.split('.');
+			var index = oidArray[oidArray.length - 1];
+			param.oid = param.walk.oid + '.' + index;
+			break;
+		}
+	}
+}
+
+function change_per_second(last, next) {
```

Comment:
Since this is not used and we likely won't add setTimeout at this point, you can remove it